alpine-rails-nginx/etc/nginx/nginx.conf

```diff
@@ -76,6 +66,7 @@ http {
 
   more_set_headers "X-Content-Type-Options: nosniff";
   more_set_headers "X-Frame-Options: deny";
+  more_set_headers "Content-Security-Policy: frame-ancestors 'none'"
```

Comment:
@mbj I am tempted to just remove the header altogether and revisit it when/if it becomes a widely deployed standard. It sounds like it's still half-baked.